### ISSUE TITLE
Redesign the "mod" icon in data: \ui\gametypes

### ISSUE THREAD

**Issue by PluffiexD** [teeworlds/teeworlds#2342]
I came across with this icon a lot of times and in my opinion it doesn't match with the other gametype icons, so I reworked it and gave it some blue touch.

new:
![new](https://user-images.githubusercontent.com/59153978/71331553-0410ef80-2533-11ea-9bfd-6c1e1fa1dc45.png)

old:
![mod](https://user-images.githubusercontent.com/59153978/71331572-1723bf80-2533-11ea-8e48-b1c132d72ec4.png)

ingame:
![ingame](https://user-images.githubusercontent.com/59153978/71331596-34f12480-2533-11ea-821c-643390d64917.png)

**Comment by AssassinTee** [comment]
i wonder if we could give the modders (on serverside) a bigger set of icons

**Comment by Dune-jr** [comment]
> i wonder if we could give the modders (on serverside) a bigger set of icons

Not really, because there is no protocol for the server to select a gametype icon, the client infers it from the gametype string. I know some other games provide a set of icons to use freely by mods, but it doesn't really apply here.
I feel like some people don't realize that they don't need to change the code to distribute a mod icon.

**Comment by oy** [comment]
@PluffiexD could you provide the .svg for it? So we can put it in the archive.

**Comment by Fisico** [comment]
I like the new one way better but the cutout of the spanner looks odd.. Maybe rotate the cutout like the current one, so it looks like a real spanner?
